src/imagefileops.cpp

```diff
@@ -18,6 +18,11 @@ namespace capypdf::internal {
 
 namespace {
 
+static std::string_view PNG_SIG("\x89PNG\r\n\x1a\n", 8);
+static std::string_view JPG_SIG("\xff\xd8\xff", 3);
+static std::string_view TIF_SIG1("II*", 3);
+static std::string_view TIF_SIG2("II*", 3);
```

Comment:
One of these should probably be `MM`.

Reply:
And also const. They don't need to be `static` since they are inside an anonymous namespace.

Reply:
Fixed!